rskj-core/src/main/java/org/ethereum/core/GenesisHeader.java

```diff
@@ -8,7 +8,7 @@
 import org.ethereum.util.ByteUtil;
 import org.ethereum.util.RLP;
 
-public class GenesisHeader extends BlockHeader {
+public class GenesisHeader extends BlockHeaderV0 {
```

Comment:
why did you use BlockHeaderV0 instead of BlockHeader?

Reply:
We can make Genesis behave like a V0 since hard fork activation is >0 in testnet and mainnet